variables.tf

```diff
@@ -112,7 +106,7 @@ variable "skip_en_cos_auth_policy" {
 
 variable "cos_integration_enabled" {
   type        = bool
-  description = "Set to `true` to connect a Cloud Object Storage service instance to your Event Notifications instance to collect events that failed delivery. If set to false, no failed events will be captured."
+  description = "Set to `true` to connect a Cloud Object Storage service instance to your Event Notifications instance to collect the messages which are failed to delivered. If set to false, no failed messages will be captured."
```

Comment:
"which are failed to delivered" is bad english. What was there before is better - why are you changing?